packages/ember-simple-auth/config/ember-try.js

```diff
@@ -14,21 +14,6 @@ module.exports = function() {
     return {
       useYarn: true,
       scenarios: [
-        {
-          name: 'ember-3.0',
```

Comment:
I believe we already spoke about this, but just to make sure - we have to drop the ember 3.0 in order to use this qunit version?

Qunit 5 is the one compatible with embroider, right?

Reply:
ember-qunit 4 does not support ember 4. I have upgraded ember-qunit to 5 to be able to support ember-4.

Reply:
Got it 🚀